README.md

```diff
@@ -1 +1,107 @@
-# atlas_starter_java
\ No newline at end of file
+# MongoDB Atlas Java Sample Project
+
+This repository contains an example application that connects
+to a MongoDB instance using the Java MongoDB Driver. You can
+use this example application as a starting point for a Java
+application that connects to MongoDB Atlas.
+
+## Prerequisites
+
+To build and run this project, you will need IntelliJ IDEA Community
+Edition for MacOS, Windows, or Linux. You can download it 
+[here](https://www.jetbrains.com/idea/).
+
+It is also helpful, but not necessary, to have a working installation of
+[Git](https://git-scm.com/downloads) version control.
+
```

Comment:
It could be useful adding JVM version requirement here as well. I think you have it in your "Notes" section, but it seems appropriate as a prerequisite.

Reply:
Good point, I'll just move both the Maven/Java requirements up here and explain that Maven comes as part of IntelliJ.